content/en/tutorials/importing-first-model-and-animation.md

```diff
@@ -0,0 +1,150 @@
+
+![][asset-pack-contents-image]
+
+## Importing the model FBX
+
+We are interested in the model file to begin with.
+
+Let's open the model folder and drag the FBX file into our project assets panel.
+
+This will import the file into our project and PlayCanvas Editor will create materials, container, render assets, and most importantly a template asset.
+
+![][importing-model-fbx-image]
+
+If you are importing FBXs into an existing project, please make sure you have the following settings ticked in the Project Settings -> Asset Tasks.
+
+![][asset-tasks-settings-image]
+
+## Adding the model Template to the scene
+
+The template asset (which are known as prefabs in other game engines) contains the full Entity hierarchy model and is used to add the model to the scene.
+
+We can do this multiple ways:
+
+Dragging it into the 3D viewport will add it as a child of the current selected Entity.
+
+![][template-add-scene-view-image]
+
+Dragging it onto an Entity in the hierarchy view will add it as a child of that Entity.
+
+![][template-add-on-entity-image]
+
+Dragging it into the empty area of the hierarchy view will add it as child of the top most Entity in the scene.
+
+![][template-add-scene-hierarchy-area-image]
+
+And finally we have the Template context menu options when right clicking on an Entity.
+
+![][template-add-context-menu-image]
+
+Now that we have it in the scene we can see it's untextured. In this case the textures were not embedded into the FBX file and have been supplied separately.
+
+If they were embedded in the FBX, when we import it would also create texture assets and they would already be assigned to the materials.
+
+You can see the texture assets being created here and also the materials with textures assigned to them.
+
+![][embedded-textures-fbx-example-image]
+
+## Adding the texture to the model
+
+Back to our Kenney Character, let's upload one of the textures from the skins folder and apply that to the material of the model.
+
+![][importing-texture-image]
+
+We can do this by either dragging the texture asset to the material slot.
+
+![][texture-to-material-image]
+
+Or we can use the 'edit' button on the material slot and find it in the assets panel.
+
+![][material-pick-texture-image]
+
+Fantastic! We now have a fully textured model in the scene, ready to be animated
+
+## Importing the animation FBX
+
+As mentioned earlier, the animations are in a separate FBX so let's import the run animation into the Editor.
+
+![][importing-animation-fbx-image]
+
+This will create a number of assets which include the animations. As that's all we need, we can delete the other assets that were created.
+
+![][delete-animation-files-image]
+
+From here, we can preview the animation by selecting it in the assets panel and in the inspector. We will also need to select a preview asset to apply the animation to. In this case, it will be the template asset created for the model FBX import earlier.
```

Comment:
```suggestion
From here, we can preview the animation by selecting it in the assets panel and in the inspector. We will also need to select a preview asset to apply the animation to. In this case, it will be the template asset created for the model FBX imported earlier.
```